src/Chief.App/MainWindow.axaml.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Avalonia.Interactivity;
+using Chief.App.Models;
+using Chief.App.Services;
+using Chief.App.Views;
+using FluentAvalonia.Core;
+using FluentAvalonia.UI.Controls;
+using FluentAvalonia.UI.Media.Animation;
+using FluentAvalonia.UI.Windowing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Chief.App;
+
+public partial class MainWindow : AppWindow
+{
+    public MainWindow()
+    {
+        InitializeComponent();
+
+        // not working in Linux
+        // TitleBar.ExtendsContentIntoTitleBar = true;
```

Comment:
So should I keep this?

Reply:
I've not tested in Windows, maybe you can uncomment it and have a try.  FluentAvalonia has'nt documented `ExtendsContentIntoTitleBar` but the same function presented in [WinUI3 API](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window.extendscontentintotitlebar?view=windows-app-sdk-1.4).